src/index.js

```diff
@@ -5,22 +5,24 @@ import App from "./App";
 import { makeServer } from "./server";
 import { BrowserRouter } from "react-router-dom";
 import { VideoCategoryProvider } from "./context/videoCategorized";
-import { FilterContextProvider  } from "./context/filter-context";
-
+import { FilterContextProvider } from "./context/filter-context";
+import { UserDetailContextProvider } from "./context/userDetail-context";
+import PlaylistModal from "./component/PlaylistModal/PlaylistModal"
 // Call make Server
```

Comment:
you can have all context files in a single index.js then where ever you need context you can import index.js

Reply:
Thank you, Rajat for pointing out that Done created the index.js  for context